src/app.js

```diff
@@ -7,7 +7,7 @@ import { mock } from './middlewares/validators/socialLogin-mock';
 
 const app = express();
 
-if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
+if (process.env.NODE_ENV === 'test' || process.env.NODE_ENV === 'development') {
```

Comment:
Is this change necessary?

Reply:
> Is this change necessary?

no, it's not necessary I was testing how the article test is passing which is magic and I realized that test never return any response you can even kill something in test and still running which is kind of magic.  maybe Christian can take his time and explain how things are running